### ISSUE TITLE
local.php.dist file mismatch

### ISSUE THREAD

**Issue by arhimede** [dotkernel/api#200]
the file local.php.dist from the repo 
is very different then the one online at api.dotkernel.net 

(local-4.1.0.php )

Let's have a final local.php file

**Comment by alexmerlin** [comment]
Not sure how `local.php` looks right now on `api.dotkernel.net`.
The last 2 modifications on `local.php.dist` were made on:
* `Mar 23, 2023`: Issue 174: Major code cleanup.
* `Aug 18, 2023`: Code quality related changes.

The diff between the last 2 commits is [here](https://github.com/dotkernel/api/commit/097834ff11fd7809cd4d6699e1f3d0489afb2834?diff=split#diff-2a493235cee7986e9a15eba3c4b8bc43ff6e9d22eeeb718ab1f27fbccb624899) and it contains only `phpcs` related modifications.

Can you post a diff between the latest `local.php.dist` and the `local.php` that is currently used on `api.dotkernel.net`?

**Comment by arhimede** [comment]
attached is the current file on api.dk

- the header with a comment 
- use different namespaces 
- PDO section 
- documentation section
[local-4.1.0.php.txt](https://github.com/dotkernel/api/files/12410563/local-4.1.0.php.txt)

**Comment by alexmerlin** [comment]
At first look, the attached version is way too out-of-sync with the repo version, containing key/values that were removed long ago (e.g.: `authentication->pdo`, `use Mezzio\Cors\Configuration\ConfigurationInterface;`)

In order to clarify things, I'll go through each item:
1. `the header with a comment`: the header comment has been removed because [README.md](https://github.com/dotkernel/api/blob/4.0/README.md?plain=1#L65) instructs developers to duplicate `config/autoload/local.php.dist` as `config/autoload/local.php`
2. `use different namespaces`: not sure what you mean because there is no namespace in neither the repo version nor the attached version. Instead, there are a some differences in the `use`s:

* the attached version contains `use Mezzio\Cors\Configuration\ConfigurationInterface;` which has been moved to [config/autoload/config/autoload/cors.local.php.dist](https://github.com/dotkernel/api/blob/4.0/config/autoload/cors.local.php.dist#L8)
* the repo version contains `use Doctrine\Common\Cache\PhpFileCache;` (also used in the attached version, but as FQCN instead of `use`) and `use Api\App\Entity\EntityListenerResolver;` added in March, this year.

3. `PDO section`: as I mentioned above, PDO section has been removed almost a year ago, more exactly in [this commit](https://github.com/dotkernel/api/commit/be054a308ecf677007da263afacc2ba8fdc457df#diff-2a493235cee7986e9a15eba3c4b8bc43ff6e9d22eeeb718ab1f27fbccb624899) after we made authentication use Doctrine vs the secondary PDO connection
4. `documentation section`: that section has been removed back in 2021, when we moved our documentation from Swagger to Postman (in [this commit](https://github.com/dotkernel/api/commit/e358cf7281bdca2c42980523d8d80aab419a340f#diff-2a493235cee7986e9a15eba3c4b8bc43ff6e9d22eeeb718ab1f27fbccb624899))
